rasa_core_sdk/endpoint.py

```diff
@@ -9,35 +9,43 @@
 from flask import Flask, jsonify, request
 from flask_cors import CORS, cross_origin
 from gevent.pywsgi import WSGIServer
-
 from rasa_core_sdk.executor import ActionExecutor
 
 DEFAULT_SERVER_PORT = 5055
 
 logger = logging.getLogger(__name__)
 
 
+def action_arg(action):
+    if "/" in action:
+        raise argparse.ArgumentTypeError(
+            'Invalid actions format. Actions file should be a python module and passed with module notation (e.g. '
+            'directory.actions).')
```

Comment:
Ok I know this was already merged, but i'd like this error message to take the argument they passed, e.g. `bot/actions` and say something like "Did you mean bot.actions?"